.pre-commit-config.yaml

```diff
@@ -95,13 +95,15 @@ repos:
         name: 'go-unit-tests'
         entry: core/scripts/pre-commit-hooks/run-go-unit-tests.sh
         files: '\.go$'
+        exclude: 'nvidia_gpu_stats*'
```

Comment:
why exclude go-unit-tests and go-vet in the nvidia_gpu_stats package? the vendor/ subdirectory is already excluded